### ISSUE TITLE
"Can you obtain security clearance?" and "TS/SCI" Questions

### ISSUE THREAD

**Issue by thomHayner** [feder-cr/Auto_Jobs_Applier_AIHawk#326]
I was watching as the program auto selected dropdown options to common questions and I noticed that it chooses "No" for the question "Can you obtain a security clearance?"

I imagine this is similar to the US/EU Work Authorization and Visa questions.

It is a common question in Engineering, Software Development, Sciences, Security and other fields, it should be added to the program.

I'm going to look into the code and see where a PR could be submitted, but if anyone is more familiar with the repo and can do so that would be helpful.

**Comment by feder-cr** [comment]
@thomHayner any update?

**Comment by thomHayner** [comment]
@feder-cr working on it, is it really just as simple as 1 field to the resume file and updating the gpt prompts?

I haven't tested it yet, I'm not really sure how to...maybe if I set my job search query to "Clearance Jobs" instead of "Software Developer"?  I'm going to try that with Hawaii as the location (90% of software jobs in Hawaii require a US security clearance)

I made the following updates:
- plain_text_resume.yaml
  - added a variable named ```security_clearance_eligibile``` to ```work_preferences```
- job_application_profile.py
  - added type for ```security_clearance_eligibile```
- gpt.py
  - modified instructions for answering ```work_preferences``` questions

 ...it looks like everything else is largely automated...is my assessment correct?  is that all I should need to do?
